Modules/@babylonjs/react-native-windows/windows/CMakeLists.txt

```diff
@@ -14,6 +14,11 @@ set(CMAKE_CXX_STANDARD_REQUIRED ON)
 set(BABYLON_REACT_NATIVE_DIR "${CMAKE_CURRENT_LIST_DIR}/../../react-native")
 set(BABYLON_NATIVE_WINDOWS_STORE_USE_SWAPCHAINPANEL 1)
 
+if (NOT GRAPHICS_API OR GRAPHICS_API STREQUAL "")
+    set (GRAPHICS_API "D3D11")
+    message("No GRAPHICS_API specified. Defaulting to D3D11.")
+endif()
+

```

Comment:
why do you need this? doesn't Babylon Native default to D3D11 if it's not set for Windows?